examples/platform/silabs/uart.cpp

```diff
@@ -20,6 +20,7 @@
 #include "MatterShell.h" // nogncheck
 #endif
 #include <cmsis_os2.h>
+#include <lib/support/CodeUtils.h>
```

Comment:
Is this change intended?

Reply:
Interesting... should have been in my previous PR, but yes. This is to use the ArraySize() maccro.
The fact that this builds without it means that something else is including it but I'd rather have the include in here.